packages/suite/src/actions/settings/walletSettingsActions.ts

```diff
@@ -84,3 +86,19 @@ export const getLastUsedFeeLevel = () => (_: Dispatch, getState: GetState) => {
     if (selectedAccount.status !== 'loaded') return;
     return settings.lastUsedFeeLevel[selectedAccount.account.symbol];
 };
+
+export const setBitcoinUnits = (units: PROTO.AmountUnit): WalletSettingsAction => ({
```

Comment:
```suggestion
export const setAmountUnit = (units: PROTO.AmountUnit): WalletSettingsAction => ({
```
let's keep consistent naming